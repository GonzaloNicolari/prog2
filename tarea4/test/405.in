# prueba para en_orden_avl, array_a_avl
arreglo_a_avl 0 .
arreglo_a_avl 1 (1,q) .
arreglo_a_avl 2 (1,q) (3,a) .
arreglo_a_avl 3 (1,q) (3,a) (5,f) .
arreglo_a_avl 4 (1,q) (3,a) (5,f) (6,z) .
arreglo_a_avl 15 (1,q) (3,a) (5,f) (6,z) (7,b) (8,c) (9,d) (10,e) (11,f) (12,g) (13,h) (14,i) (15,j) (16,k) (17,l).
en_orden_avl
insertar_en_avl (10,mmm)
en_orden_avl
insertar_en_avl (20,nnn)
en_orden_avl
insertar_en_avl (30,ooo)
en_orden_avl
insertar_en_avl (13,mo)
insertar_en_avl (18,mq)
insertar_en_avl (15,mof)
insertar_en_avl (25,no)
insertar_en_avl (5,hhh)
insertar_en_avl (3,eee)
insertar_en_avl (8,hm)
insertar_en_avl (9,hz)
insertar_en_avl (12,mz)
insertar_en_avl (16,hz)
insertar_en_avl (11,my)
insertar_en_avl (14,md)
insertar_en_avl (40,sss)
insertar_en_avl (17,ge)
insertar_en_avl (35,saa)
en_orden_avl
Fin
