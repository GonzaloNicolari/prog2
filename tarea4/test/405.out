1>#  prueba para en_orden_avl, array_a_avl.
2>
3>
1 
4>
3 
1 
5>
1 5 
3 
6>
6 
1 5 
3 
7>
1 5 7 9 11 13 15 17 
3 8 12 16 
6 14 
10 
8>
9>Insertado.
10>10 
11>Insertado.
12>10 20 
13>Insertado.
14>10 20 30 
15>Insertado.
16>Insertado.
17>Insertado.
18>Insertado.
19>Insertado.
20>Insertado.
21>Insertado.
22>Insertado.
23>Insertado.
24>Insertado.
25>Insertado.
26>Insertado.
27>Insertado.
28>Insertado.
29>Insertado.
30>3 5 8 9 10 11 12 13 14 15 16 17 18 20 25 30 35 40 
31>Fin.
